src/index.js

```diff
@@ -76,15 +77,17 @@ export default async function microbundle(options) {
 	options.name =
 		options.name || options.pkg.amdName || safeVariableName(options.pkg.name);
 
-	const jsOrTs = async filename =>
+	const getFileType = async filename =>
 		resolve(
 			cwd,
 			`${filename}${
 				(await isFile(resolve(cwd, filename + '.ts')))
 					? '.ts'
 					: (await isFile(resolve(cwd, filename + '.tsx')))
 						? '.tsx'
-						: '.js'
+						: (await isFile(resolve(cwd, filename + '.re')))
```

Comment:
i would rewrite this `getFileType` to mutliple if+return pairs instead of nesting so many ternaries

Reply:
Will do, wasn’t too sure on what to refactor, could use some cleanup indeed